### DISCUSSION TITLE
Need SteamDB Users information

### DISCUSSION THREAD

**Discussion by sethu3375** [SteamRE/SteamKit#1141] [General]
Is it possible to get user level information from steamDB? From a backend API or scraping?

**Comment by yaakov-h** [comment]
You'd have to ask SteamDB, but they don't expose an API to my knowledge and don't allow scraping either.

**Comment by xPaw** [comment]
What do you need SteamDB for, use Valve's apis.

https://steamapi.xpaw.me/#IPlayerService/GetSteamLevel
